src/mapblock.cpp

```diff
@@ -222,25 +222,20 @@ s16 MapBlock::getGroundLevel(v2s16 p2d)
 {
 	if(isDummy())
 		return -3;
-	try
+	s16 y = MAP_BLOCKSIZE-1;
+	for(; y>=0; y--)
```

Comment:
In future changes, please correct the code style of each touched (/modified) line according to the rules specified here: https://dev.minetest.net/Code_style_guidelines

Thank you :)

Reply:
Fixed.

Reply:
Can the initialization of `y` go in the `for`?
Also spaces around operators and after `for` and `if`.

Reply:
I knew there was code that I touched and had messed up spacing. Thanks. Fixed.